src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1,6 +1,67 @@
-%h4.list-group.mb-4 Comments & Request History
-.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
-  = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
+.row
+  .col-md-8#overview
+    .mb-4#description-text
+      - if bs_request.description.present?
+        = render partial: 'webui/shared/collapsible_text', locals: { text: bs_request.description }
+      - else
+        %i No description set
+    .comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
+      = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
+    .comment_new.mt-3
+      = render partial: 'webui/comment/new', locals: { commentable: bs_request }
+  .col-md-4#side-links
+    %ul.list-unstyled
+      - unless package_maintainers.empty?
+        %li
+          %i.fas.fa-info-circle.text-info
+          = render partial: 'maintainer_link', locals: { maintainers: package_maintainers }
+      - if bs_request.superseding.present?
+        %li
+          %i.fas.fa-plus-circle.text-primary
+          Supersedes
+          - bs_request.superseding.each do |supersed|
+            = link_to supersed['number'], number: supersed['number']
+      - if bs_request.accept_at.present?
+        %li
+          %i.fas.fa-exclamation-circle.text-danger
+          - if BsRequest::FINAL_REQUEST_STATES.include?(bs_request.state)
+            Auto-accept was set to
+            %span.fuzzy-time{ title: "#{l bs_request.accept_at}" }
+            = succeed '.' do
+              = l bs_request.accept_at, format: :only_date
+          - elsif bs_request.accept_at.past?
+            This request will be automatically accepted when it enters the 'new' state.
+          - else
+            This request will be automatically accepted in #{time_ago_in_words(bs_request.accept_at)}.
+      - if bs_request.accepted_reviews.present?
+        .px-0.px-md-1.py-0
```

Comment:
I think you go a bit overboard with all the utility classes, I'm not sure what you are trying to achieve by moving this pixel by pixel? Make it for instance not aligned with the side-links anymore etc. Are you sure you still need this?

![Screenshot from 2022-08-05 17-08-43](https://user-images.githubusercontent.com/514785/183107189-6f469dc8-4f05-418b-b241-a37c596f9466.png)

Reply:
It's updated